From 4d8176d1e9ef5cbd2b3fcacd9b9a4c8e482fb424 Mon Sep 17 00:00:00 2001
From: Jan Schneider <jan@horde.org>
Date: Tue, 6 Sep 2016 16:15:10 +0200
Subject: [PATCH] [jan] SECURITY: Fix XSS via data:text/html content of form
 action and xlink attributes (Reported by Liuzhu <fantasy7082@hotmail.com>).

---
 framework/Text_Filter/package.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/framework/Text_Filter/package.xml b/framework/Text_Filter/package.xml
index 231e35ec2d7..b507a3ff94b 100644
--- a/framework/Text_Filter/package.xml
+++ b/framework/Text_Filter/package.xml
@@ -33,7 +33,7 @@
  </stability>
  <license uri="http://www.horde.org/licenses/lgpl21">LGPL-2.1</license>
  <notes>
-* 
+* [jan] SECURITY: Fix XSS via data:text/html content of form action and xlink attributes (Reported by Liuzhu &lt;fantasy7082@hotmail.com&gt;).
  </notes>
  <contents>
   <dir baseinstalldir="/" name="/">
@@ -1111,7 +1111,7 @@ Initial release as a PEAR package
    <date>2016-03-21</date>
    <license uri="http://www.horde.org/licenses/lgpl21">LGPL-2.1</license>
    <notes>
-* 
+* [jan] SECURITY: Fix XSS via data:text/html content of form action and xlink attributes (Reported by Liuzhu &lt;fantasy7082@hotmail.com&gt;).
    </notes>
   </release>
  </changelog>
